src/network/networkd-link.c

```diff
@@ -2163,6 +2163,19 @@ static int link_set_can(Link *link) {
                         return log_link_error_errno(link, r, "Could not append IFLA_CAN_RESTART_MS attribute: %m");
         }
 
+        if (link->network->can_triple_sampling > -1) {
```

Comment:
not that it matters, but usually we write these tri-state checks as `a >= 0` rather than `a > -1`...